### PR TITLE
 #31 : finish build yolo9000 detection model class

### PR DESCRIPTION
**Pull request by ssaru**
재원님 입원으로 스프린트 딜레이가 예상되므로, 제가 model을 작성해서 PR 날립니다.

### CODE REVIEW

File: model/model.py [hunk 1]
```diff
@@ -1,22 +1,140 @@
...
+
+        self.device = device
+        self.input_size = input_size
+        self.num_coordinates = 5
+        self.num_prior_boxes = num_prior_boxes
+        self.num_classes = num_classes
+        self.output_shape = (self.num_coordinates + self.num_classes) * self.num_prior_boxes
+
+        self.layer1 = nn.Sequential(
+            nn.Conv2d(3, 32, kernel_size=3, stride=1, padding=1),
```
Comment (visionNoob) [inline model/model.py:30]: batch norm을 적용하므로 bias = 0 옵션이 들어가야 할 것 같습니다
(제가 새로 PR한 코드에 적용했습니다).

File: model/model.py [hunk 2]
```diff
@@ -1,22 +1,140 @@
...
+            nn.BatchNorm2d(32),
+            nn.LeakyReLU(),
```
Comment (visionNoob) [inline model/model.py:32]: (https://pytorch.org/docs/stable/nn.html)
`torch.nn.LeakyReLU(negative_slope=0.01, inplace=False)`

LeakyReLU의 negative_slope 초기값이 0.001 인데 YOLO에서는 0.1을 써서 이 부분도 수정했습니다!

File: model/model.py [hunk 3]
```diff
@@ -1,22 +1,140 @@
...
+            nn.BatchNorm2d(1024),
+            nn.LeakyReLU(),
+            nn.Conv2d(1024, 512, kernel_size=1, stride=1, padding=0),
+            nn.BatchNorm2d(512),
+            nn.LeakyReLU(),
+            nn.Conv2d(512, 1024, kernel_size=3, stride=1, padding=1),
+            nn.BatchNorm2d(1024),
+            nn.LeakyReLU())
+
+        self.layer7 = nn.Sequential(
```
Comment (visionNoob) [inline model/model.py:100]: YOLO9000 논문을 보면 

```
Training for detection. We modify this network for detection by 
removing the last convolutional layer and instead adding on three 
3 × 3 convolutional layers with 1024 filters each followed by a final 
1 × 1 convolutional layer with the number of outputs we need for detection.
```
이라는 부분이 있습니다. 
layer7에 3개의 conv2d가 있어야 하는게 아닌가요?
